### DISCUSSION TITLE
Limit the range can selected based on start date in useRangePicker

### DISCUSSION THREAD

**Discussion by vanntrong** [adobe/react-spectrum#6595] [Q&A]
Hello,

I would like to ask if there is any way to set the maxDate prop based on the `value.start`. Like in other packages, when the user selects a `start value`, the days that are out of range will be disabled. How can I do that?

Thank you.

**Comment by snowystinger** [comment]
This isn't something we currently support, but it seems useful. I'll convert this to an issue for further discussion.
